.github/workflows/pythonbuild.yml

```diff
@@ -248,6 +250,8 @@ jobs:
       - uses: actions/checkout@v4
         with:
           fetch-depth: 0
+      - name: 'Clear action cache'
+        uses: ./.github/actions/clear-action-cache
```

Comment:
why did we need this?  can you add a comment?

Reply:
updated it, sandbox has disk pressure